ytypes/node_test.go

```diff
@@ -808,9 +809,9 @@ func treeNodesEqual(got, want []*TreeNode) error {
 		if !match {
 			paths := []string{}
 			for _, g := range got {
-				paths = append(paths, fmt.Sprintf("< %s | %#v >", proto.CompactTextString(g.Path), g))
```

Comment:
Did `proto.CompactTextString` get deprecated?

Reply:
I changed to the new proto library the checker recommended. I couldn't find this function in https://godoc.org/google.golang.org/protobuf/encoding/prototext, so I used this.

Reply:
Sure - I'm happy with this. Looks like this might be part of the protobuf refactoring that's been going on.